app/views/layouts/dashboard/_head_panel.html.erb

```diff
@@ -22,8 +22,8 @@
             <a href="blank.html">
               <%= image_tag('user.png', :class => 'wd-80 rounded-circle', :all => '') %>
             </a>
-            <h6 class="logged-fullname"><%= current_user.name %></h6>
-            <p><%= current_user.email%></p>
+            <h6 class="logged-fullname"><%= current_user.try(:name) || current_admin.name %></h6>
```

Comment:
pq o current_user usa o try(:name) e o current_admin não?

Reply:
O comportamento do try é o seguinte: ele tenta aplicar o método sobre o objeto (no caso, aplicar `name` sobre o `current_user` e, caso o objeto seja `nil`, retorno `nil` em vez de levantar a exeption `Undefined Method for Nil Class`. Como se chega nessa página somente tendo sessão de user ou admin e já se viu que não há sessão de user, é regra que vai ter sessão de admin, por isso não precisa do `try`. Acha que fica melhor com try nos dois?

Reply:
Acho que não, era só pra enteder pq um tinha e o outro não.